server/utils/formUtils.ts

```diff
@@ -86,3 +86,7 @@ export function convertKeyValuePairToCheckBoxItems<T>(
     }
   })
 }
+
+export function validPostcode(potentialPostcode: string) {
```

Comment:
This should just be for the postcode area (i.e. SW1, WS13 etc etc)

Reply:
Ok I'll update the function